### PR TITLE
Meetings service changed to sort agenda items before action items

### PR DESCRIPTION
**Pull request by LouisDG6**
## Proposed changes

This PR proposes to list agenda_owning_participant_id nulls last 

## Checklist

- [x] Are the issues being addressed linked to this PR?
- [x] Do all commit messages start with the issue number?
- [x] Are all code changes sufficiently tested?
- [ ] Are there screenshots for UI changes?

### CODE REVIEW

File: api/src/services/__tests__/meetingsService.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ describe('meetingsService', () => {
         [{ id: meetingId, starts_at: startsAt }]
       );
       mockQuery(
-        'select `meeting_notes`.`id` as `id`, `note_text`, `sort_order`, `authoring_participant_id`, `agenda_owning_participant_id` from `meeting_notes` inner join `participants` on `participants`.`id` = `meeting_notes`.`authoring_participant_id` where `participants`.`meeting_id` = ? order by `agenda_owning_participant_id` asc, `sort_order` asc, `meeting_notes`.`created_at` asc',
+        'select `meeting_notes`.`id` as `id`, `note_text`, `sort_order`, `authoring_participant_id`, `agenda_owning_participant_id` from `meeting_notes` inner join `participants` on `participants`.`id` = `meeting_notes`.`authoring_participant_id` where `participants`.`meeting_id` = ? order by `agenda_owning_participant_id is null` asc, `agenda_owning_participant_id` asc, `sort_order` asc, `meeting_notes`.`created_at` asc',
```
Comment (davidvandusen) [inline api/src/services/__tests__/meetingsService.ts:47]: I just tried to run this query against my database and it failed. How was this tested?

**Review by davidvandusen — CHANGES_REQUESTED** [review]

File: api/src/services/__tests__/meetingsService.ts (hunk 1 above)
Comment (davidvandusen) [reply api/src/services/__tests__/meetingsService.ts:47]: The error was:

```
Unknown column 'agenda_owning_participant_id is null' in 'order clause'
```

It looks like the way that it's specified, the whole string `'agenda_owning_participant_id is null'` is quoted as if it's a column name. There must be a different syntax for defining raw order by statements.

Comment (LouisDG6) [reply api/src/services/__tests__/meetingsService.ts:47]: Thanks for the comment. Will make the appropriate changes.

**Comment by pthan1** [comment]
PR https://github.com/OpenTree-Education/rhizone-lms/pull/339 was merged to address Issue https://github.com/OpenTree-Education/rhizone-lms/issues/316

**Pull request closed without merging**